recipes-robot/recipes-images/images/opentrons-reference-minimal-image.bb

```diff
@@ -0,0 +1,48 @@
+inherit core-image
+
+SUMMARY = "Opentrons Embedded Linux Reference Minimal Image"
```

Comment:
can we call this like `opentrons-ot3-image` or something both in the filename and the summary, might as well begin as we mean to go on